script/build-task.js

```diff
@@ -33,6 +33,21 @@ const MODULE_JSON =
 				"type": "module",
 				"manifest": "https://raw.githubusercontent.com/TheGiddyLimit/plutonium-next/master/module.json",
 			},
+			{
+				"name": "midi-qol",
+				"type": "module",
+				"manifest": "https://gitlab.com/tposney/midi-qol/raw/master/package/module.json",
+			},
+			{
+				"name": "dae",
+				"type": "module",
+				"manifest": "https://gitlab.com/tposney/dae/raw/master/package/module.json",
+			},
+			{
+				"name": "times-up",
+				"type": "module",
+				"manifest": "https://gitlab.com/tposney/times-up/raw/master/package/module.json",
+			},
```

Comment:
I checked, and not only is using core `StatusEffect`s a bit buggy with DAE (unless I screwed something up), but, more importantly, it can't be hijacked by DFCE. If you set e.g. `StatusEffect CUSTOM Frightened (CE)` while DFCE adds its CEs to the core list (or replaces them), and then turn off that setting and try to import the item to an actor, the AE breaks and defaults to the core 'Asleep' status (because it's the first alphabetically).

So we need DFCE to be a hard dependency (because who actually runs a campaign without ever using _any_ conditions).

```suggestion
			},
			{
				"name": "dfreds-convenient-effects",
				"type": "module",
				"manifest": "https://github.com/DFreds/dfreds-convenient-effects/releases/latest/download/module.json",
			},
```

We also therefore need to actually use DFCE for conditions (`StatusEffect CUSTOM ConditionName (CE)`), as well as sort out the force-DFCE-settings thing. But that's a TODO for a later date for now, and a README note will suffice even if we go live before that's finished.

Reply:
Everything else is a 👌 from me though 👍